combat-trainer.lic

```diff
@@ -970,7 +971,7 @@ class SpellProcess
     release_cyclics if data['cyclic']
     command = 'pre'
     command = data['skill'] != 'Targeted Magic' ? 'pre' : 'tar' if data['skill']
```

Comment:
The times I've done sorcery, I've listed it as `- skill: sorcery` and it seemed to work alright. I don't have any sorcery spells to test with, but in theory any user specified attributes should be kept